Makefile

```diff
@@ -37,6 +40,20 @@ stop-airflow2:
 	@ echo "$(BUILD_PRINT)Stopping the AirFlow services"
 	@ docker-compose --file ./airflow2/docker-compose.yml --env-file ../.env down
 
+start-tika: build-volumes start-storage
```

Comment:
Tika starts once the elk starts, not sure we need this make target.
If we need to control tika separately, I propose then to move it out of the elk docker-compose into a dedicated compose file.

Reply:
Tika does not start when ELK starts.